src/main/resources/org/aion/wallet/ui/components/partials/SyncStatus.fxml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<?import javafx.scene.control.Label?>
+<?import javafx.scene.control.ProgressBar?>
+<?import javafx.scene.layout.AnchorPane?>
+
+<AnchorPane layoutX="150.0" prefHeight="46.0" prefWidth="200.0" xmlns="http://javafx.com/javafx/8.0.121" xmlns:fx="http://javafx.com/fxml/1" fx:controller="org.aion.wallet.ui.components.partials.SyncStatus">
+    <ProgressBar fx:id="syncProgressBar" layoutX="86.0" layoutY="13.0" prefWidth="200.0" />
```

Comment:
![screenshot from 2018-03-30 23-01-34](https://user-images.githubusercontent.com/37304045/38152107-b9131000-346e-11e8-9ab1-0f49a722f50e.png)
Will the sync bar extend all the way to the end of the writing? If now wou should conside displaying only the first 2 time units (in this case 55 days 15 hours)